src/locales/zh_CN/battle.ts

```diff
@@ -73,6 +75,14 @@ export const battle: SimpleTranslationEntries = {
   "statHarshlyFell": "{{pokemonNameWithAffix}} 的 {{stats}} 大幅降低了！",
   "statSeverelyFell": "{{pokemonNameWithAffix}} 的 {{stats}} 极大幅降低了！",
   "statWontGoAnyLower": "{{pokemonNameWithAffix}} 的 {{stats}} 已经无法再降低了！",
+  "statRoseMultiple": "{{pokemonNameWithAffix}}'s {{stats}} rose!",
+  "statSharplyRoseMultiple": "{{pokemonNameWithAffix}}'s {{stats}} sharply rose!",
+  "statRoseDrasticallyMultiple": "{{pokemonNameWithAffix}}'s {{stats}} rose drastically!",
+  "statWontGoAnyHigherMultiple": "{{pokemonNameWithAffix}}'s {{stats}} won't go any higher!",
+  "statFellMultiple": "{{pokemonNameWithAffix}}'s {{stats}} fell!",
+  "statHarshlyFellMultiple": "{{pokemonNameWithAffix}}'s {{stats}} harshly fell!",
+  "statSeverelyFellMultiple": "{{pokemonNameWithAffix}}'s {{stats}} severely fell!",
+  "statWontGoAnyLowerMultiple": "{{pokemonNameWithAffix}}'s {{stats}} won't go any lower!",
```

Comment:
```suggestion
  "statRoseMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都提高了！",
  "statSharplyRoseMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都大幅提高了！",
  "statRoseDrasticallyMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都极大幅提高了！",
  "statWontGoAnyHigherMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都已经无法再提高了！",
  "statFellMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都降低了！",
  "statHarshlyFellMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都大幅降低了！",
  "statSeverelyFellMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都极大幅降低了！",
  "statWontGoAnyLowerMultiple": "{{pokemonNameWithAffix}}的 {{stats}} 都已经无法再提高了！",
```
cn translation, added a "都" to indicate plural